docs/README.md

```diff
@@ -0,0 +1,350 @@
+# Selenium.Axe for .NET
```

Comment:
Is it intentional that this content hasn't been removed from the root level README? I think the root level readme should probably keep a basic Getting Started inline, but it can probably lose the detailed reference docs if there's going to be a separate page for it

Reply:
This was intentional.  
Wanted to get team buy in before shortening the readme.

Can do the shortened readme in this pull request as well, assuming there is no big pushback on adding the docify docs.